### PR TITLE
Add breadcrumb navigations for FAQ categories

### PR DESCRIPTION
**Pull request by jaroslav-kubicek**
<url>LiveURL: https://kiwicomsmart-faq-hmtnlsaqep.now.sh
</url>
</url>
</url>
</url>

### CODE REVIEW

**Comment by git-toni** [comment]
This looks good!
Will wait until graphql issue is approved to merge so we can test it with real data :)

File: src/StaticFAQ/Breadcrumb.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+          .breadcrumb-icon {
+            margin: 0 8px;
+          }
+        `}
+      </style>
+    </div>
+  );
+};
+
+Breadcrumb.defaultProps = {
```
Comment (git-toni) [inline src/StaticFAQ/Breadcrumb.js]: We didn't use this `defaultProps` syntax so far. What about adding it at the argument destructuring at Line 19?

Comment (jaroslav-kubicek) [reply src/StaticFAQ/Breadcrumb.js]: unfortunately, flow works only with `defaultProps`

**Comment by jaroslav-kubicek** [comment]
updated, cc @git-toni

**Comment by jaroslav-kubicek** [comment]
there is an issue for that, I would leave it for newcomers #90 @git-toni

**Comment by git-toni** [comment]
What about the back button at the breadcrumbs? Should we raise an issue for that? @jaroslav-kubicek 
image

**Pull request merged**